daq/host.py

```diff
@@ -114,7 +115,7 @@ def __init__(self, runner, gateway, target, config):
     @staticmethod
     def make_runid():
         """Create a timestamped runid"""
-        return '%06x' % int(time.time())
+        return '%06x' % int(time.time() * 10e6)
```

Comment:
I think 10e6 might be a bit aggressive... in what circumstances does this need to be unique at the us level?

Reply:
I wanted to make it more unique which was why I added the multiplier. Maybe I can do 10e3 instead.